Lib/ufo2ft/outlineOTF.py

```diff
@@ -801,14 +804,24 @@ def setupOtherTables(self):
     def setupTable_glyf(self):
         """Make the glyf table."""
 
+        if self.convertCubics:
+            from cu2qu.pens import Cu2QuPen
+
         self.otf["loca"] = newTable("loca")
         self.otf["glyf"] = glyf = newTable("glyf")
         glyf.glyphs = {}
         glyf.glyphOrder = self.glyphOrder
 
         for name in self.glyphOrder:
             pen = TTGlyphPen(self.allGlyphs)
+            if self.convertCubics:
+                pen = Cu2QuPen(pen, self.cubicConversionError,
+                               reverse_direction=True)
             self.allGlyphs[name].draw(pen)
+            if self.convertCubics:
+                # travel back through the hairy nest of conversion pens.
+                # why can't there just be a single pen protocol...
+                pen = pen.pen.pen._outPen.pen.pen
```

Comment:
As much as I like how well this code illustrates my problem with pens, @adrientetar pointed out that it would be nicer to just remember the original TTGlyphPen in a separate variable and call glyph() from that.